### PR TITLE
[Search] Update release date in changelog

### PR DESCRIPTION
**Pull request by ShivangiReja**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

File: sdk/search/Azure.Search.Documents/CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 11.4.0 (Unreleased)
+## 11.4.0 (2022-09-06)
```
Comment (jsquire) [inline sdk/search/Azure.Search.Documents/CHANGELOG.md:3]: Should this be 9/7 since Monday is a holiday?

**Review by jsquire — APPROVED** [review]

File: sdk/search/Azure.Search.Documents/CHANGELOG.md (hunk 1 above)
Comment (ShivangiReja) [reply sdk/search/Azure.Search.Documents/CHANGELOG.md:3]: September 6th is on Tuesday.

Comment (jsquire) [reply sdk/search/Azure.Search.Documents/CHANGELOG.md:3]: Understood, but we normally push the release schedule for a holiday... but I just found the dates announcement and it looks like Tuesday is still the day.   Ignore me.  😄

**Pull request merged**